amcl/package.xml

```diff
@@ -16,8 +16,7 @@
     <url>http://wiki.ros.org/amcl</url>
     <author>Brian P. Gerkey</author>
     <author>contradict@gmail.com</author>
-    <maintainer email="davidvlu@gmail.com">David V. Lu!!</maintainer>
-    <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
+    <maintainer email="dhood@osrfoundation.org">D. Hood</maintainer>
```

Comment:
Please add author fields to credit previous maintainers (same below)

Reply:
they're already there

Reply:
sorry, "to credit previous maintainers". sure.